ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -121,6 +126,62 @@ internal class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
         return false
     }
 
+    func handleCapacitorHttpRequest(_ urlSchemeTask: WKURLSchemeTask) {
+        var urlRequest = urlSchemeTask.request
+        let url = urlRequest.url!
+        var targetUrl = url.absoluteString
+            .replacingOccurrences(of: CapacitorBridge.httpInterceptorStartIdentifier, with: "")
+
+        // Only replace first occurrence of the scheme
+        if let range = targetUrl.range(of: InstanceDescriptorDefaults.scheme) {
+            targetUrl = targetUrl.replacingCharacters(in: range, with: "https")
```

Comment:
We are hardcoding the scheme to https here, but the original url could be http.
We are going to need to also store the original scheme as part of the modified url for iOS and doing the replacement to the original scheme, or only proxy https urls (and document that the proxy only works for https urls)